event-sourcing/event-repository/event-repository-jdbc/src/main/java/uk/gov/justice/services/eventsourcing/repository/jdbc/eventlog/EventLogConverter.java

```diff
@@ -52,7 +49,7 @@ public EventLog createEventLog(final JsonEnvelope envelope, final UUID streamId,
                 eventMetadata.name(),
                 envelope.metadata().asJsonObject().toString(),
                 extractPayloadAsString(envelope),
-                clock.now());
+                eventMetadata.created().get());
```

Comment:
We should check there is a value present and throw an exception with an explanation if it is not there.